features/attached_enable.feature

```diff
@@ -28,6 +28,25 @@ Feature: Enable command behaviour when attached to an Ubuntu Pro subscription
             | xenial  |
             | bionic  |
 
+    @series.xenial
+    @series.bionic
+    @uses.config.machine_type.lxd.container
+    Scenario Outline: Enable cc-eal with --access-only
+        Given a `<release>` machine with ubuntu-advantage-tools installed
+        When I attach `contract_token` with sudo
+        When I run `pro enable cc-eal --access-only` with sudo
+        Then I will see the following on stdout:
+        """
+        One moment, checking your subscription first
+        Updating package lists
+        Skipping installing packages: ubuntu-commoncriteria
+        CC EAL2 access enabled
```

Comment:
do you think it would be beneficial to include in the tests another command making sure that installing the package works after enabling with --access-only?

Reply:
good idea!